ui/DesignSystem/Component/SourceBrowser/FileView/PeerStatus.svelte

```diff
@@ -0,0 +1,28 @@
+<script>
+  import { Avatar, Icon } from "../../../Primitive";
```

Comment:
We can use absolute imports now.
```suggestion
  import { Avatar, Icon } from "ui/DesignSystem/Primitive";
```